plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -211,7 +224,7 @@ private static String quotedTableName(SchemaTableName name)
         return quotedName(name.getSchemaName()) + "." + quotedName(name.getTableName());
     }
 
-    private static String quotedName(String name)
+    public static String quotedName(String name)
```

Comment:
this doesn't seem to have external usage, revertible?

Reply:
nice catch!